src/gt4py/gt2_src_manager.py

```diff
@@ -70,7 +70,7 @@ def remove_gt_sources():
 
 def has_gt_sources():
     return os.path.isfile(
-        os.path.join(gt_config.GT_INCLUDE_PATH, "gridtools", "common", "defs.hpp")
```

Comment:
So that was the line that was causing the error when only GT2 was installed?

Reply:
Yes!